content/docs/tasks/telemetry/metrics-logs/index.md

```diff
@@ -38,8 +38,9 @@ as the example application throughout this task.
     spec:
       value: "2" # count each request twice
       dimensions:
-        source: source.service | "unknown"
-        destination: destination.service | "unknown"
+        reporter: conditional((context.reporter.kind | "inbound") == "outbound", "client", "server")
+        source: source.name | "unknown"
```

Comment:
should we use `source.workload.name` and `destination.workload.name` here instead?

or `source.labels["app"]` and `destination.labels["app"]` ?

same question below for the `newlog` changes.

with these changes, the data drops to be something like:
```
destination="details-v1",instance="172.17.0.12:42422",job="istio-mesh",message="twice the fun!",reporter="client",source="productpage-v1"
```

Reply:
Thanks.. changed.